<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLCDNumber" name="lcdNumber_stat">
     <property name="digitCount">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLCDNumber" name="lcdNumber_start">
     <property name="digitCount">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLCDNumber" name="lcdNumber_end">
     <property name="digitCount">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLCDNumber" name="lcdNumber_finish">
     <property name="digitCount">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
